### ISSUE TITLE
Change in "Functional" label in PSI HTML for WebMO

### ISSUE THREAD

**Issue by jrschmidt2** [psi4/psi4webmo#10]
We are trying to get more consistency in our labeling of fields across different comp. chem. programs.

Any objections to simply re-labeling the "Functional" field to "DFT Functional" in psi.html?

**Comment by loriab** [comment]
This is in the menu where users set up their job? Fine by me. 

If a fairly broad list of QC energy terms would ever be useful to your consistency efforts, let me know.

**Comment by jrschmidt2** [comment]
Yes, this is the job options page.  Let's start with just the "DFT Functional" change.  Many of the other options are consistently named.  In this case it was just PSI that was the outlier.

(The other glaring inconsistency is between "Theory" and "Method".  We use both in various engines with seemingly no logic!  It would be interesting to have a discussion about which might be preferred.)
